Mathlib/Algebra/Polynomial/Splits.lean

```diff
@@ -427,6 +427,23 @@ theorem mem_lift_of_splits_of_roots_mem_range [Algebra R K] {f : K[X]}
   obtain ⟨b, hb, rfl⟩ := Multiset.mem_map.1 hP
   exact Subring.sub_mem _ (X_mem_lifts _) (C'_mem_lifts (hr _ hb))
 
+/--
+A polynomial of degree `2` with a root splits.
+-/
+theorem splits_of_natDegree_eq_two {f : Polynomial K} {x : L} (h₁ : f.natDegree = 2)
```

Comment:
Is it a lot work to generalize this to degree `n` polynomials with `n-1` roots?

Reply:
It should not be that harder. I'll try in another PR.